src/esp/assets/ResourceManager.h

```diff
@@ -658,24 +659,21 @@ class ResourceManager {
    * If both parent and drawables are provided, add the mesh to the
    * scene graph for rendering.
    * @param info The @ref AssetInfo for the mesh, already parsed from a file.
+   * @param creation How to instance the render asset, or nullptr if not
+   * instancing.
    * @param parent The @ref scene::SceneNode to which the mesh will be added
-   * as a child.
+   * as a child. See also creation->isRGBD and creation->isSemantic. nullptr if
```

Comment:
There's an important subtlety here: currently, the way the user chooses RGBD vs Semantic is by passing the appropriate parent and drawables (from either the RGBD or Semantic scene graph). But I'm also requiring the user to specify isRGBD and isSemantic in the creation struct (so that I can record the choice in the replay data).

Eventually, we could refactor this interface so that the user just specifies the creation struct, and then we choose the correct parent and drawables on the backend.